app/models/user_auth_token.rb

```diff
@@ -58,7 +58,7 @@ def self.is_suspicious(user_id, user_ip)
     return false if ips.empty? # first login is never suspicious
 
     user_location = login_location(user_ip)
-    ips.none? { |ip| user_location == login_location(ip) }
+    ips.none? { |ip| distance(user_location, login_location(ip)) < MAX_SUSPICIOUS_DISTANCE_KM }
```

Comment:
I would go with a hidden site setting here instead, cause it will allow us to tweak it more easily.

Additionally people who want the distance thing off can just set it to 0.

Reply:
Hm.. sounds like a great idea. Thank you!